packages/worker/src/sdk/auth/auth.ts

```diff
@@ -73,7 +73,7 @@ export const reset = async (email: string) => {
  * Perform the user password update if the provided reset code is valid.
  */
 export const resetUpdate = async (resetCode: string, password: string) => {
-  const { userId } = await redis.checkResetPasswordCode(resetCode)
+  const { userId } = await redis.passwordReset.getResetPasswordCode(resetCode)
```

Comment:
This appears to be the only place in the codebase that fetches the password reset code, and it immediately discards the `info` portion. If the `info` isn't used, should we get rid of it?